### PR TITLE
DOC Better description of inset locator and colorbar

### PR DESCRIPTION
**Pull request by NelleV**
## PR Summary

This provides a better title and description of the example, as well as uses fig.colorbar instead of plt.colorbar.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by NelleV** [comment]
My pleasure!

File: examples/axes_grid1/demo_colorbar_with_inset_locator.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
 """
-================================
-Demo Colorbar With Inset Locator
-================================
+=======================================================
+Controling the position of colorbars With Inset Locator
```
Comment (QuLogic) [inline examples/axes_grid1/demo_colorbar_with_inset_locator.py]:
```suggestion
Controlling the position of colorbars with Inset Locator
```
Not sure; should "Inset Locator" be lower-case, too?

Comment (timhoffm) [reply examples/axes_grid1/demo_colorbar_with_inset_locator.py]: Shouldn‘t that be something like „using inset_axes“. The term „inset locator“ does not appear in the example at all.

**Comment by NelleV** [comment]
@timhoffm agreed. I changed it.

Can whoever merge squash this when merging? I am happy to squash manually as well

**Comment by NelleV** [comment]
Thanks @dstansby !

**Pull request merged**